File: SAMPLE/CS/language.cs
// -- IMPORTS

using LINGUI;

// -- TYPES

namespace LINGUI
{
    public class LANGUAGE : BASE_LANGUAGE
    {
        // -- CONSTRUCTORS

        public LANGUAGE(
            ) : base()
        {
        }

        // -- INQUIRIES

        public virtual string GameOver(
            )
        {
            return "";
        }

        // ~~

        public virtual string Welcome(
            string first_name,
            string last_name
            )
        {
            return "";
        }

        // ~~

        public virtual string Pears(
            int count
            )
        {
            return "";
        }
    }
}

File: SAMPLE/DART/language.dart
// -- IMPORTS

import "base_language.dart";
import "genre.dart";
import "plurality.dart";
import "translation.dart";

// -- TYPES

class LANGUAGE extends BASE_LANGUAGE
{
    // -- CONSTRUCTORS

    LANGUAGE(
        ) : super()
    {
    }

    // -- INQUIRIES

    String GameOver(
        )
    {
        return "";
    }

    // ~~

    String Welcome(
        TRANSLATION first_name_translation,
        TRANSLATION last_name_translation
        )
    {
        return "";
    }

    // ~~

    String Pears(
        TRANSLATION count_translation
        )
    {
        return "";
    }
}
